app/controllers/join.js

```diff
@@ -1,14 +1,63 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
+import { APPS } from '../constants/urls';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { toastNotificationTimeoutOptions } from '../constants/toast-notification';
 
 export default class JoinController extends Controller {
   @service router;
   @service login;
   @service featureFlag;
+  @tracked Chaincode = 'Generate Chaincode';
```

Comment:
Why `c` is in capital, it doesn't follow the convention of the app

Reply:
Done With changes